smoketests/README.rst

```diff
@@ -0,0 +1,145 @@
+==================
+Kitsune smoketests
+==================
+
+This directory holds Kitsune client-based smoke tests. We use the same
+infrastructure that `Fjord uses <https://github.com/mozilla/fjord/>`_.
+
+These tests use the following things:
+
+1. py.test
+2. pytest-mozwebqa
+3. pytest-xdist
+4. selenium
+
+
+Setting up to run the tests
+===========================
+
+I suggest using a different virtual environment for these tests than the
+rest of Kitsune so you're not mixing requirements::
+
+    $ mkvirtualenv kitsune-smoketests
+    $ pip install -r requirements.txt
```

Comment:
Is mkvirtualenv a real thing? Is that part of virtualenv wrapper? I'd vote that we simply leave off the command part of this and say something like 

> I suggest using a different virtual environment for these tests than the
> rest of Kitsune so you're not mixing requirements. The requirements are
> listed in requirements.txt.